tools/github-page-script-injection/injectScript.js

```diff
@@ -32,7 +32,7 @@ let scriptContent = fs.readFileSync(scriptFilePath, 'utf8');
 const connectionString = 'InstrumentationKey=1ae9e7ce-18f1-4e14-8fc0-acbf0ed28895;IngestionEndpoint=https://eastus-8.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/';
 let iKeyForCfgSync = "1ae9e7ce-18f1-4e14-8fc0-acbf0ed28895";
 //let cfgUrl = "https://js.monitor.azure.com/beta/ai.config.1.cfg.json";
-let cfgUrl = "https://js.monitor.azure.com/nightly/ai_test.config.1.cfg.json";
+let cfgUrl = "https://js.monitor.azure.com/nightly/ai_test.config.1.0-nightly3.cfg.json";
```

Comment:
Right now it's probably `1.0-nightly3` (because the version gets flattened from 1.0.0 to 1.0 and 1) but it probably should be listed as `1-nightly3`

Reply:
just changed it! Thanks Nev!